gulp/atom.js

```diff
@@ -1,13 +1,33 @@
-var gulp = require( 'gulp' );
-var atom = require( 'gulp-atom' );
-
-gulp.task( 'atom', function () {
-	return atom( {
-		srcPath: './compile',
-		releasePath: './build',
-		cachePath: './cache',
-		version: 'v0.26.0',
-		rebuild: false,
-		platforms: [ 'darwin-x64' ]
-	} );
+var gulp = require('gulp');
+var atom = require('gulp-atom');
+var symlink = require('gulp-sym');
+var fs = require('fs');
+var runSequence = require('run-sequence');
+
+var s = {
+	srcPath: './compile',
+	releasePath: './build',
+	cachePath: './cache',
+	version: 'v0.26.0',
+	rebuild: false,
+	platforms: ['darwin-x64'] // Set at the first place your dev platform for correct linking of resources
+}
+var appPath = s.releasePath + '/' + s.version + '/' + s.platforms[0] + '/Electron.app/Contents/Resources/app';
```

Comment:
It's path was only works on OSX?
And path separator `/` also platform dependent. should use `path.join`